fixtures/group/advance-group.json

```diff
@@ -0,0 +1,26 @@
+{
+  "type": "group",
+  "userId": "nhi",
+  "groupId": "libraries",
+  "traits": {
+    "name": "Initech",
+    "industry": "Technology",
+    "employees": 329,
+    "plan": "enterprise",
+    "total billed": 830,
+    "id": "{{id}}"
+  },
+  "context": {
+    "app": {
```

Comment:
Should this be in the fixture? As each library will collect this differently. Or is the intention that the harness override this?

Reply:
good point, was just an oversight. Will change it to something other than `app`.